src/ResourceManager/Profile/Commands.Profile/help/Send-Feedback.md

```diff
@@ -38,7 +38,7 @@ Response.
 Please enter your email if you are interested in providing follow up information:
 
 your@email.com
-```
```

Comment:
@sankethka I think that the changes to the backticks (removing them on this line and adding them above on line 25) should be reverted so that the output from the cmdlet is in the code block. This is the pattern we try to follow with all cmdlets - the commands run in the example and any output from those commands should be in the code block, and any additional notes you want to make about what is being executed should be outside the code block and below the example. How does that sound?